crates/holochain/src/core/ribosome/host_fn/get_link_details.rs

```diff
@@ -147,4 +157,34 @@ pub mod slow_tests {
         }
         assert!(remove_happened);
     }
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn get_link_details_local_only() {
```

Comment:
I couldn't come up with a way to test for the opposite, get links from network. Everyone is an authority in the small test scenarios and network calls are prevented for authorities. I'd need to set the arc somehow.

Reply:
Agree, that's a general limitation of our ability to test the cascade logic from any code built on top of it